SPECS/pytorch/CVE-2022-1941.patch

```diff
@@ -0,0 +1,352 @@
+# Patch generated by Archana Choudhary <archana1@microsoft.com>
+# Source: https://github.com/protocolbuffers/protobuf/commit/55815e423bb82cc828836bbd60c79c1f9a195763
```

Comment:
Can you please mention how did you arrive at this patch. The NVD page says the vulnerability present in any version previous to 3.18.3. But this patch is merged to 3.16.2 and it doesn't have detailed description too.

Can you please elaborate how this patch fixes the vulnerability ?

Reply:
This PR thread explains the fix: https://github.com/microsoft/azurelinux/pull/9850